src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -60,12 +73,15 @@ describe("App", () => {
         await waitFor(() => {
             expect(screen.getByText("Offers Management")).toBeInTheDocument();
             expect(screen.getAllByText("Guy in the background")).toHaveLength(2);
+        }, {
+            timeout: 1000,
         });
     });
 
     test("Loads Empty Offers", async () => {
-        companyOffersService.fetchCompanyOffers = jest.fn(() =>  Promise.resolve([]));
-
+        companyOffersService.fetchCompanyOffers.mockImplementationOnce(() =>  new Promise((resolve) =>
+            resolve([])
+        ));
```

Comment:
You can return a "resolved" promise instantly like this

```suggestion
        companyOffersService.fetchCompanyOffers.mockImplementationOnce(() =>  Promise.resolve([])));
```